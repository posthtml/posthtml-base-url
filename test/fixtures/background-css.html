<head>
  <style>
    .test {
      background-image: url('image.jpg');
      background: url('image.jpg');
      background-image: url('https://preserve.me/image.jpg');
      background: url('https://preserve.me/image.jpg');
    }

    .test-2 {
      background-image: url("image.jpg");
      background: url("image.jpg");
      background-image: url("https://preserve.me/image.jpg");
      background: url("https://preserve.me/image.jpg");
    }

    .test-3 {
      background-image: url(image.jpg);
      background: url(image.jpg);
      background-image: url(https://preserve.me/image.jpg);
      background: url(https://preserve.me/image.jpg);
    }
  </style>
</head>

<body style="background: url('image.jpg')">
  <table style="background: url(image.jpg)">
    <tr>
      <th style="background-image: url('image.jpg')">Header</th>
    </tr>
    <tr>
      <td style="background-image: url(image.jpg)">Cell</td>
    </tr>
  </table>
</body>
