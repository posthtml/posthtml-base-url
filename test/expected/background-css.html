<head>
  <style>.test {
      background-image: url('https://example.com/image.jpg');
      background: url('https://example.com/image.jpg');
      background-image: url('https://preserve.me/image.jpg');
      background: url('https://preserve.me/image.jpg');
    }

    .test-2 {
      background-image: url("https://example.com/image.jpg");
      background: url("https://example.com/image.jpg");
      background-image: url("https://preserve.me/image.jpg");
      background: url("https://preserve.me/image.jpg");
    }

    .test-3 {
      background-image: url(https://example.com/image.jpg);
      background: url(https://example.com/image.jpg);
      background-image: url(https://preserve.me/image.jpg);
      background: url(https://preserve.me/image.jpg);
    }</style>
</head>

<body style="background: url('https://example.com/image.jpg')">
  <table style="background: url(https://example.com/image.jpg)">
    <tr>
      <th style="background-image: url('https://example.com/image.jpg')">Header</th>
    </tr>
    <tr>
      <td style="background-image: url(https://example.com/image.jpg)">Cell</td>
    </tr>
  </table>
</body>
